Gems/Atom/RPI/Code/Include/Atom/RPI.Public/ViewportContext.h

```diff
@@ -98,6 +119,12 @@ namespace AZ
             //! Alternatively, connect to ViewportContextNotificationsBus and listen to ViewportContextNotifications::OnViewportDpiScalingChanged.
             void ConnectDpiScalingFactorChangedHandler(ScalarChangedEvent::Handler& handler);
 
+            using UintChangedEvent = AZ::Event<uint32_t>;
```

Comment:
suggestion: Including the type in the event name might not be ideal, prefer the actual value it's referring to, so this might be `RenderSettingChangedEvent` perhaps? Or have a different type for `Vsync` and `RefreshRate`?